test-suite/handwritten-src/cpp/extended_record.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "extended_record_base.hpp"
+
+namespace testsuite {
+	
+	struct ExtendedRecord : public ExtendedRecordBase
+	{
+		using ExtendedRecordBase::ExtendedRecordBase;
+	};
+}
```

Comment:
The file doesn't end in a newline (mouse over the symbol), which is recommended practice and can cause all kinds of problems if not followed.